### DISCUSSION TITLE
Hiding in-game UI should also hide the "press/hold for menu" button

### DISCUSSION THREAD

**Discussion by LittleEndu** [ppy/osu#13437] [Ideas]
> Opening #8110 as a discussion

When you press <kbd>Shift</kbd>+<kbd>Tab</kbd>, all in-game UI is hidden except for the "press for menu" X at the bottom right corner. It fades to low opacity but is still clearly visible when not surrounded by any other UI.

**Comment by peppy** [comment]
The reasoning for this is to allow users without a keyboard to still exit the game. I guess at the point of pressing a key binding to hide the interface you're guaranteed to have a keyboard at that point, but not sure this will be the case into the future.

Something to think about for sure.

**Reply by LittleEndu** [reply]
I think if the X faded to 0% instead when UI is hidden would already be enough to provide the accessibility required from it. Also, aren't HUD visibility options also exposed in settings meaning shift+tab isn't the only way to hide the UI. Just more things to consider.

**Comment by JayFlowGT** [comment]
Could this just be an option in settings?

Gameplay > HUD > Completely fade out the menu button during gameplay

**Reply by Theighlin** [reply]
This is gonna be alleviated by https://github.com/ppy/osu/pull/26128 if it gets in.

**Comment by WalmartWhiteCat** [comment]
I think it would be cool if the X acted like the Touchscreen mod. If Touchscreen is on, the X is visible, if Touchscreen is off, then the X is gone. I _also_ think it would be cool if you could adjust the opacity and choose if the X is always visible or only if Touchscreen is on. Also, maybe allow us to choose where the X is on screen? That would be nice for modes like Mania and Catch for people who tap close to the far bottom of the screen, like me.
